GUIs/ScriptRunnerPyQt5_GUI/scripFunc/scripWINONLY.py

```diff
@@ -524,13 +549,195 @@ def calladdusrtosys():
         calladdusrtosys()
 
     def remusr(self):
-        pass
+        class rmvusrfrosys(QDialog, Ui_rmvusrogru):
```

Comment:
Codacy found an issue: ['QDialog' may be undefined, or defined from star imports: PyQt5.QtWidgets, threading (F405)](https://app.codacy.com/manual/vipersniper0501/CP_Scripts2/pullRequest?prid=5927260)